patroni/ha.py

```diff
@@ -187,6 +195,28 @@ def bootstrap(self):
                 return 'trying to ' + msg
             return 'waiting for leader to bootstrap'
 
+    def bootstrap_standby_leader(self):
+        """ If we found 'standby' key in the configuration, we need to bootstrap
+            not a real master, but a 'standby leader', that will take base backup
+            from a remote master and start follow it.
+        """
+        config_value = json.dumps(
```

Comment:
There should be nither `/config` nor `/leader` key in DCS until bootstrap is done.